.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+image:
+    file: .gitpod.dockerfile
+ports:
+    - port: 3000
+      onOpen: open-preview
+tasks:
+    - init: >
+        bundle install --without production &&
+        cp config/database.yml.gitpod config/database.yml &&
+        mysql -e "SET @@global.sql_mode=(SELECT REPLACE(@@global.sql_mode, 'ONLY_FULL_GROUP_BY', ''));"; &&
+        mysql -e "CREATE DATABASE plots;"; &&
```

Comment:
I believe the issue is here, there's a `;` and then `&&`, this is a syntax error.

Reply:
Hmm, after this fix we still see it:

```
gitpod /workspace/plots2 $  HISTFILE=/workspace/.gitpod/cmd-0 history -r; {
> bundle install --without production && cp config/database.yml.gitpod config/database.yml && mysql -e "SET @@global.sql_mode=(SELECT REPLACE(@@global.sql_mode, 'ONLY_FULL_GROUP_BY', ''));" && mysql -e "CREATE DATABASE plots;"; && cp db/schema.rb.example db/schema.rb && rake db:setup && yarn install
bash: syntax error near unexpected token `&&'
gitpod /workspace/plots2 $ 
gitpod /workspace/plots2 $ } && {
bash: syntax error near unexpected token `}'
gitpod /workspace/plots2 $ passenger start
bash: passenger: command not found
gitpod /workspace/plots2 $ 
gitpod /workspace/plots2 $ }
bash: syntax error near unexpected
```

But, actually there's a `;` on the next line too, so I'll try removing that.